### PR TITLE
link fixes in documentation after migration

### PR DESCRIPTION
**Pull request by kleash**
- Links changed to relative path instead of absolute
- Readme format fix

Signed-off-by: Shubham Aggarwal <shubham6300@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,32 +69,32 @@ These are the exported functions available for WASM Chaincode Developers. Most o
...
-- ```installedChaincodes``` give back all installed wasm chaincodes
+WASMCC have three functions available to initiate transaction from clients, namely, `create`, `invoke` and `installedChaincodes` functions for a transaction
+- `create` accepts wasm chaincode name, wasm chaincode in form of wasm binary or zip or hex value and the function parameters for init function of wasm chaincode
+    - `create` invokes init function of wasm chaincode
+    - `create` stores the chaincode in state on successful init invocation
+- `invoke` accepts wasm chaincode name, function to invoke and parameters(to be passed to wasm)
+    - `invoke` retrieves the wasm chaincode bytes from state and execute it in wasm vm
+    - `invoke` dynamically invokes the function from wasm chaincode whose name it accepted as a parameter initially. Also it will send number of transaction parameters available to wasm function
+    - wasm chaincode can retrieves the parameter using exported `getParameters` function
+    - wasm chaincode can returns the result and the result using __return_result` function and. For success it should return 0 and for error it should return -1
```
Comment (MHBauer) [inline README.md]: missing a single `

**Review by MHBauer — COMMENTED** [review]
commit message should say something like

- use repo-relative linking
- use single backtick instead of triple backtick for inline verbatim text.

File: README.md (hunk 1 above)
Comment (kleash) [reply README.md]: Thanks for pointing out. Fixed

**Comment by kleash** [comment]
> commit message should say something like
> 
> * use repo-relative linking
> * use single backtick instead of triple backtick for inline verbatim text.

fixed

File: README.md [hunk 2]
```diff
@@ -189,13 +189,13 @@ If successful, do a query again and balance of account1 should be 110 now.
...
+More information here : [link](tools/wasm-pusher)
 
 ## Run Unit test
 
- - Compile sample rust chaincode at ```sample-wasm-chaincode/chaincode_example02/rust/src/lib.rs```  to wasm binary : [instructions](https://github.com/kleash/fabric-chaincode-wasm/tree/master/sample-wasm-chaincode/README.md)
- - Go to ```wasmcc``` directory
- - Give command ```go test```
+ - Compile sample rust chaincode at `sample-wasm-chaincode/chaincode_example02/rust/src/lib.rs`  to wasm binary : [instructions](sample-wasm-chaincode/README.md)
+ - Go to `wasmcc` directory
+ - Give command `go test`
```
Comment (MHBauer) [inline README.md:198]: a TODO for later. should attempt to refer to the makefile.

**Review by MHBauer — APPROVED** [review]
LGTM

File: README.md (hunk 2 above)
Comment (kleash) [reply README.md:198]: @MHBauer good point. opened an issue here for same: #25

**Pull request merged**